### PR TITLE
Feature/27 school page archives

### PR DESCRIPTION
**Pull request by lraivio**
This is built on top of #58, so that should be merged first.

This PR:
- refactors school details
- adds archives, school types and school fields, languages and genders to school details

### CODE REVIEW

**Comment by YoYuUm** [comment]
Good PR, I have sent to flowdock images for the rows using the default one.

File: app/stores/SchoolStore.js [hunk 1]
```diff
@@ -170,12 +46,125 @@ SchoolStore.dispatchToken = AppDispatcher.register(function(payload) {
...
+  };
 }
 
 function _receiveSchools(schools) {
   _.each(schools, _receiveSchool);
 }
 
+function _sortByYears(list) {
+  return _.sortByOrder(list, ['end_year', 'begin_year'], [false, false]);
+}
```
Comment (YoYuUm) [inline app/stores/SchoolStore.js]: Try to leave last or out of the list items without beginning nor ending dates.

**Comment by YoYuUm** [comment]
You will need to refactor this branch and remove all commits before 798b50a, including it.

**Comment by lraivio** [comment]
I replaced this PR with #62 to avoid massive merge conflicts.

**Pull request closed without merging**